modules/i3s/src/i3s-loader.js

```diff
@@ -0,0 +1,131 @@
+/* global fetch, TextDecoder,  __VERSION__ */ // __VERSION__ is injected by babel-plugin-version-inline
+
+const VERSION = typeof __VERSION__ !== 'undefined' ? __VERSION__ : 'latest';
+
+import {Ellipsoid} from '@math.gl/geospatial';
```

Comment:
In other loaders, we tend to keep the loader file quite small, and import a parse function from `lib/parse-i3s.js`